browser/brave_tab_helpers.cc

```diff
@@ -27,7 +28,8 @@ void AttachTabHelpers(content::WebContents* web_contents) {
 #endif
   // Add tab helpers here unless they are intended for android too
 #endif
-
+  brave_shields::TrackingProtectionHelper::CreateForWebContents(
```

Comment:
this should be moved above the `endif` per the comment

Reply:
Resolved